src/java.base/share/classes/java/security/spec/PSSParameterSpec.java

```diff
@@ -94,17 +88,26 @@
     private final int trailerField;
 
     /**
-     * The {@code TrailerFieldBC} constant as defined in PKCS#1
+     * The {@code TrailerFieldBC} constant as defined in the PKCS #1 standard.
      *
      * @since 11
      */
     public static final int TRAILER_FIELD_BC = 1;
 
     /**
      * The PSS parameter set with all default values
+     * @deprecated This field uses the default values defined in the PKCS #1
+     *         standard. Some of these defaults are no longer recommended due
+     *         to advances in cryptanalysis -- see the
+     *         <a href="https://tools.ietf.org/rfc/rfc8017.txt">PKCS#1 v2.2</a>
+     *         standard for more details. Thus, it is recommended to create
+     *         a new {@code PSSParameterSpec} with the desired parameter values
+     *         using
+     *         {@link #PSSParameterSpec(String, String, AlgorithmParameterSpec, int, int) PSSParameterSpec}.
```

Comment:
I think it would be more clear to see the full signature of the ctor that you are recommending be used instead, so I would change these 2 lines to:

`using the {@link #PSSParameterSpec(String, String, AlgorithmParameterSpec, int, int)} constructor.`

Reply:
Makes sense.